third_party/wabt/src/interp/jit/jit-backend.cc

```diff
@@ -44,6 +45,13 @@ namespace interp {
 static const uint8_t kContextReg = SLJIT_S0;
 static const uint8_t kFrameReg = SLJIT_S1;
 
+using f32Function2Param =
```

Comment:
This could go to an inline file to reduce the size of this file. See
`wabt/include/wabt/interp/jit/jit-imath32-inl.h`